src/test/Components/CreateImageWizard/CreateImageWizard.test.js

```diff
@@ -760,7 +760,9 @@ describe('Step File system configuration', () => {
     );
     await user.click(manuallyConfigurePartitions);
 
-    const addPartition = await screen.findByTestId('file-system-add-partition');
+    const addPartition = screen.getByRole('button', {
+      name: /add partition/i,
+    });
```

Comment:
Tests shouldn't use `getBy` but only `findBy` why did you need to revert this back?

Reply:
change it back to findby